src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -1175,6 +1175,22 @@ If you encounter problems with configuring your log forwarder, try these trouble
     sudo apt install td-agent-bit=1.8.12
     ```
   </Collapser>
+  
+  <Collapser
+    className="freq-link"
+    id="many-cfg-files-in-tmp"
+    title="Many configuration files get created under /tmp/nr_fb_config*, or Fluent Bit is constantly restarted"
+  >
+    The Infrastructure Agent takes the `*.yml` files under the `logging.d` configuration directory, generates a Fluent Bit configuration file out of them (following the naming `nr_fb_config*`) and launches a Fluent Bit instance with this resulting configuration file. Nevertheless, this Fluent Bit process will be restarted and its associated `nr_fb_config*` regenerated if any of the following events happen:
+    * The Infrastructure Agent itself is restarted
+    * Any of the `*.yml` files under `/etc/newrelic-infra/logging.d/*.yml` change
+    * The `hostname` of the host where the Infrastructure Agent is installed changes
+    
+    We've encountered several situation in which the `hostname` changes very frequently, which can result in Fluent Bit getting constantly restarted and the temporary directory where the Fluent Bit configuration files are generated getting filled up with `nr_fb_config*` files.
+    
+    This situation is external to the Infrastructure Agent. A workaround for this is to fix the hostname reported by the Infrastructure Agent by setting the [`override_hostname`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#override-hostname) and [`override_hostname_short`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#override-hostname-short) configuration options. Note though that by using this workaround, the Infrastructure Agent will not report the DNS name anymore. 
+
```

Comment:
```suggestion
   The Infrastructure Agent takes the `*.yml` files under the `logging.d` configuration directory, generates a Fluent Bit configuration file out of them (following the naming `nr_fb_config*`) and launches a Fluent Bit instance with this resulting configuration file. The Fluent Bit process will be restarted and its associated `nr_fb_config*` will be regenerated if any of the following events happen on the host:
    * The Infrastructure Agent is restarted.
    * Any of the `*.yml` files under `/etc/newrelic-infra/logging.d/*.yml` are updated (config hot-reloading capability).
    * The `hostname` of the host where the Infrastructure Agent is installed changes.
    
    On specific environments, it might happen that the `hostname` changes very frequently, which can result in Fluent Bit getting constantly restarted and the temporary directory where the Fluent Bit configuration files are generated getting filled up with `nr_fb_config*` files. 
    To solve this issue, the hostname reported by the Infrastructure Agent can be explicitly defined by setting the [`override_hostname`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#override-hostname) and [`override_hostname_short`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#override-hostname-short) configuration options. Note that by using these settings, the agent will not report the DNS name anymore. 
    [Learn more](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#hostname-variables) about Hostname settings for the Infrastructure Agent.

```

Reply:
The above is just a suggestion, I'd let the Docs team to try to improve the context structure (maybe add headers for: Context, Problem, Solution?)